pkgs/data/fonts/open-sans/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitLab }:
+
+stdenv.mkDerivation rec {
+  pname = "open-sans";
+  version = "1.11";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "fonts-team";
+    repo = "fonts-open-sans";
+    rev = "fa55fea78526fb0356be9cdd8f92f53c92f807b9";
```

Comment:
They have a tag for this, but for some reason it didn't like it.

Reply:
```suggestion
    rev = "debian%2F1.11-1"; # URL-encoded form of "debian/1.11-1" tag
```

Reply:
Right this isn't a hierarchical structure.